packages/miniflare/test/fixtures/sites/modules.ts

```diff
@@ -2,8 +2,19 @@ import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
 import manifestJSON from "__STATIC_CONTENT_MANIFEST";
 const manifest = JSON.parse(manifestJSON);
 
-export default <ExportedHandler<{ __STATIC_CONTENT: KVNamespace }>>{
+export default <
+	ExportedHandler<{
+		__STATIC_CONTENT: KVNamespace;
+		__STATIC_CONTENT_MANIFEST?: undefined;
+	}>
+>{
 	async fetch(request, env, ctx) {
+		if (env.__STATIC_CONTENT_MANIFEST !== undefined) {
```

Comment:
Do we not only want to assert env.__STATIC_CONTENT_MANIFEST is undefined but also that it is not on the env object at all?
```suggestion
		if ("__STATIC_CONTENT_MANIFEST" in env) {
```